include/usb/usb_device.h

```diff
@@ -186,25 +186,28 @@ struct usb_request_handlers {
  * chosen configuration of the USB device.
  * The structure represent class in the core USB stack.
  */
-struct usb_cfg_data {
+struct usb_class_data {
 	/** Function for interface runtime configuration */
 	usb_interface_config interface_config;
 	/** Callback to be notified on USB connection status change */
-	void (*cb_usb_status)(struct usb_cfg_data *cfg,
+	void (*cb_usb_status)(struct usb_class_data *class_data,
 			      enum usb_dc_status_code cb_status,
 			      const uint8_t *param);
 	/** USB request handlers */
 	struct usb_request_handlers request_handlers;
-	/* Number of interfaces for this function */
+	/** Number of interfaces for this class */
 	const uint8_t num_interfaces;
```

Comment:
const uint8_t num_interfaces; can be removed? or moved to usb_if_container?

Reply:
`const uint8_t num_interfaces;` is number of `usb_if_containers` so it must stay here.

Reply:
Would it make sense to rename it too, `num_if_containers`?

Reply:
Definitely yes. I will rename it. Wondering if We should also rethink the `usb_ep_cfg_data` and make it consistent with each other.

Reply:
Maybe just usb_ep_data or usb_ep_cfg?

Reply:
I will leave endpoints for now. What I will change is:
`usb_cfg_data::interfaces -> usb_cfg_data::if_containers`
`usb_cfg_data::num_interfaces -> usb_cfg_data::num_if_containers`